gh-84867: Do not load tests from `TestCase` and `FunctionTestCase` (#100497)
python/cpython: Misc/NEWS.d/next/Library/2022-12-24-12-50-54.gh-issue-84867.OhaLbU.rst

> Do not load test cases from exact :class:`unittest.case.TestCase` and
> :class:`unittest.case.FunctionTestCase` classes.

Reviewer: Simply `unittest.TestCase` and `unittest.FunctionTestCase`. They are more known under simpler names.

And maybe mention `unittest.TestLoader`, because it is not clear who do not load them.

Like this:
```suggestion
:class:`unittest.TestLoader` no longer loads test cases from exact
:class:`unittest.TestCase` and :class:`unittest.FunctionTestCase` classes.
```
